hoot-services/src/main/java/hoot/services/controllers/grail/PullApiCommand.java

```diff
@@ -94,8 +94,7 @@ private void getApiData() {
                         ") is too large. It must be less than " + maxBboxArea + " degrees");
             }
 
-            url = replaceSensitiveData(params.getPullUrl()) +
-                    "/mapfull?bbox=" + boundingBox.toServicesString();
+            url = replaceSensitiveData(params.getPullUrl()) + "?bbox=" + boundingBox.toServicesString();
```

Comment:
Just commenting for posterity that we don't need `mapfull` to rehydrate relations, because we are now excluding coastlines/bodies of water and political boundaries that are most likely to be modeled as a relation of unclosed ways.  Also we are not exporting the area features to a geometry type (e.g. shapefile) that demands the polygon be fully rehydrated and closed.